kinoml/workflows/chembl22_cnn.py

```diff
@@ -0,0 +1,188 @@
+'''
+This script uses the pre-processed data file df_reduced.pkl that is generated from the ChEMBL22 data set.
+
+Type of target used: single-target (i.e. ligand-based model)
+Type of featurizer for the ligand: One Hot Encoding of SMILES
+Type of model: a Convolutional Neural Network (CNN) using Pytorch
+
+Please run in terminal the following command:
+$ export PYTHONPATH=$(dirname $(pwd))
+
+And check the PYTHONPATH with: 
+$ echo $PYTHONPATH
+
+'''
+# Imports
+import pandas as pd
+import numpy as np
+from pathlib import Path
+from sklearn.model_selection import train_test_split, KFold
+from sklearn.metrics import accuracy_score, confusion_matrix
+
+import torch
+import torch.nn as nn
+import torch.optim as optim
+
+from ml.models_pytorch import CNN
+from features.ligand import OneHotSMILESFeaturizer
+
+
+if __name__ == '__main__':
+
+    # Load pre-processed data
+    p = Path(__file__).resolve()
+    data = pd.read_pickle(f'{p.parents[1]}/data/df_reduced.pkl')
+
+    # Set threshold for activity
+    c = 6.3
+    print(f'Threshold at {c}')
```

Comment:
If we are aiming at providing library-like components, `print()` shouldn't be used because downstream users won't be able to change the verbosity easily (they would need to patch `sys.stdout` or weird hacks like that), so it's better if we use  `logging`.